about.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Exhibition</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="navbar">
+                <ul class="top-navbar-menu">
+                    <li><a href="">Facebook</a></li>
+                    <li><a href="">Twitter</a></li>
+                    <li><a href="">English</a></li>
+                    <li><a href="">My Page</a></li>
+                    <li><a href="">Log out</a></li>
+                </ul>
+                <div class="bottom-navbar">
+                    <a class="logo" href="index.html">Logo</a>
+                    <ul class="bottom-navbar-menu">
+                        <li><a href="about.html">About</a></li>
```

Comment:
It actually means you have to change the link,  `<li><a href="about.html">About</a></li>` to this  `<li><a href="index.html">Home</a></li>`

Reply:
the previous reviewer did not say anything about this at all. and why should I change it because we are not in the about page we are in the home page and when we click on the about it should go to the about page thanks. I did not understand why you comment about it.

Reply:
Dear @Esmatullah12 , look at it this way, after you have clicked the `about` link and are taken to the About page, while you are at the About page, it does not make sense to have `<li><a href="about.html">About</a></li>` which still links to the same About page. instead, you should have `<li><a href="index.html">Home</a></li>` which leads you back to the Home page. 

If you need more clarification, I am very available to explain further via a Zoom call.